src/routes/volume/VolumeList.js

```diff
@@ -210,13 +210,21 @@ function list({
               {restoreProgress}
               {rebuildProgress}
             </div>
-            {isEncrypted ? <Tooltip title={'Encrypted Volume'}><Icon className="color-warning" style={{ marginRight: 5, marginBottom: 2 }} type="lock" /></Tooltip> : null}
+            {isEncrypted ? <Tooltip title={'Encrypted Volume'}><Icon className="color-warning" style={{ marginLeft: 5, marginRight: 5, marginBottom: 2 }} type="lock" /></Tooltip> : null}
             {statusUpgradingEngine(record)}
             { upgrade }
-            {attchedNodeIsDown ? <Tooltip title={'The attached node is down'}><Icon className="faulted" style={{ transform: 'rotate(45deg)', marginRight: 5 }} type="api" /></Tooltip> : ''}
+            {attachedNodeIsDown && (
+              <Tooltip title={'The attached node is down'}>
+                <Icon className="faulted" style={{ transform: 'rotate(45deg)', marginRight: 5, marginLeft: 5 }} type="api" />
```

Comment:
Add some marginLeft for volume state column Icon

<img width="443" alt="Screenshot 2024-06-06 at 8 42 30 PM" src="https://github.com/longhorn/longhorn-ui/assets/5744158/a1e9adcd-ad20-487e-9194-5e41c5f4fd85">